firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -74,9 +74,11 @@ export class FirestoreBigQueryEventHistoryTracker
     if (typeof eventData === "undefined") {
       return undefined;
     }
-
     const data = traverse(eventData).map((property) => {
-      if (property instanceof firebase.firestore.DocumentReference) {
+      if (
```

Comment:
Could this not be shorted to `property.constructor?.name ===` with TypeScript?

Reply:
Unfortunately, prettier errors when using the optional operator. Updating the version brings a raft of style changes.